templates/search.html

```
@@ -44,47 +44,47 @@
                 </div>
             </nav>
         </header>
 
-        {% if enable_vsearch %}
-            {% if search_type == "keyword" %}
-                {% if total < 30 %}
-                    {% if !search.is_empty() %}
-                    <p class="suggest_vsearch">当前结果较少，建议使用 <a href="/?search={{ search }}&search_type=vsearch">语义搜索</a></p>
+        {% if search_meta.enable_vsearch %}
+            {% if search_meta.search_type == "keyword" %}
+                {% if search_meta.total < 30 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)